apps/site/src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,30 @@
+import Container from "react-bootstrap/Container";
+import Nav from "react-bootstrap/Nav";
+import Navbar from "react-bootstrap/Navbar";
+
+import "bootstrap/dist/css/bootstrap.min.css";
```

Comment:
It might be better if you included the Bootstrap line in a more global TSX file such as `layout.tsx`.